src/Sylius/Bundle/PayumBundle/Model/PaymentConfig.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+* This file is part of the Sylius package.
+*
+* (c) Paweł Jędrzejewski
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Sylius\Bundle\PayumBundle\Model;
+
+use Payum\Core\Model\PaymentConfigInterface;
+
+class PaymentConfig implements PaymentConfigInterface
```

Comment:
The dynamic payment part should be added in another PR.
If everything is good as is, it's good to merge (just requires rebase).